docker.go

```diff
@@ -481,7 +481,40 @@ func (c *DockerContainer) CopyFileFromContainer(ctx context.Context, filePath st
 	return ret, nil
 }
 
+// CopyDirToContainer copies the contents of a directory to a parent path in the container. This parent path must exist in the container first
+// as we cannot create it
```

Comment:
```suggestion
// as we can not create it
```

Reply:
https://www.grammarly.com/blog/cannot-or-can-not/#:~:text=Here%20is%20a%20quick%20summary,.%20.%20.%20but%20also.%E2%80%9D 😄